src/components/Node.js

```diff
@@ -63,10 +63,7 @@ const Node = ({ type, name, path, parentCommitmessage, level }) => {
           }
         )
       } else {
-        const fileExtension = path
-          .split('.')
-          .slice(-1)[0]
-          .toLowerCase()
+        const fileExtension = path.split('.').slice(-1)[0].toLowerCase()
```

Comment:
Automatically done by prettier, not sure how that slipped in before :)

Also, I noticed that prettier is not listed in `package.json`. The pre-commit hook was expecting it to be installed globally. I think this should be fixed as a global prettier is not used a lot AFAIK and therefore most folks do not have prettier installed globally.

Reply:
Ah, good point about prettier. I had it installed globally for some reason, and didn't notice it wasn't present as a dependency 🙄